datafusion/expr/src/logical_plan/builder.rs

```diff
@@ -400,7 +400,7 @@ impl LogicalPlanBuilder {
         Self::scan_with_filters(table_name, table_source, projection, vec![])
     }
 
-    pub fn delim_get(correlated_columns: &Vec<CorrelatedColumnInfo>) -> Result<Self> {
```

Comment:
This should be unique by (Column,DataType), there is a case that two correlated columns being the same, but with different depth => the output of this logical plan has duplicate column